ffi_internals/src/consumer.rs

```diff
@@ -111,8 +91,8 @@ fn write_primitive_conformances(consumer_dir: &str) -> Result<(), std::io::Error
         "bool", "u8", "u16", "u32", "u64", "i8", "i16", "i32", "i64", "f32", "f64",
     ]
     .iter()
-    .map(|native_type| {
-        let consumer_type = ffi_internals::consumer_type_for(native_type, false);
+    .try_for_each(|native_type| {
```

Comment:
This can switch back to a `map`. (I briefly started to have everything that could fail return errors, with the thought that I could add spans to them eventually and emit better errors that way, before I found `proc_macro_error`.)

Reply:
NM, this one can actually return `std::io::Error` so it should be doing `try_for_each`.